staging/src/k8s.io/api/apiserverinternal/v1alpha1/types.go

```diff
@@ -47,7 +47,7 @@ type StorageVersionStatus struct {
 	// The reported versions per API server instance.
 	// +optional
 	// +listType=map
-	// +listMapKey=apiserverID
+	// +listMapKey=apiServerID
```

Comment:
did this bug (referring to a field name that doesn't exist) get caught by a linter? if not, can you open an issue against the linter (I think it lives in https://github.com/kubernetes/kube-openapi) to add that check?

Reply:
opened https://github.com/kubernetes/kube-openapi/issues/205 to track